packages/@sanity/cli/jest.config.cjs

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const {createJestConfig} = require('../../../test/config.cjs')
+
+module.exports = createJestConfig({
+  displayName: require('./package.json').name,
+  testEnvironment: 'node',
+  rootDir: __dirname,
+  globalSetup: '<rootDir>/test/shared/globalSetup.ts',
+  globalTeardown: '<rootDir>/test/shared/globalTeardown.ts',
+  slowTestThreshold: 60000,
+  testTimeout: 30000,
+  setupFilesAfterEnv: ['<rootDir>/test/shared/setupAfterEnv.ts'],
```

Comment:
Nitpick – personally I'm a fan of alphabetizing lists like this. It's probably just ocd 😅

Reply:
:) Fixed